### PR TITLE
Ja localization

### PR DESCRIPTION
**Pull request by Mayuko-Sa**
Updated EN -> JA documentation translations

English texts remain in this PR, which will be translated next time.

Code comments (starts with #) are no longer translated. When changes are made to the code, code comments that were previously translated to Japanese may be reverted to English, but this is allowed.

### CODE REVIEW

**Review by kbatuigas — APPROVED** [review]
🎉

File: jekyll/_cci2_ja/runner-overview.adoc [hunk 1]
```diff
@@ -16,7 +16,7 @@ version:
 
 toc::[]
 
-NOTE: 現在、よりスケーラブルでコンテナフレンドリーなセルフホストランナー、<a data-type="xref" href="container-runner.html">コンテナランナー</a>の<strong>オープンプレビュー</strong>を行っています。 詳細については、<a data-type="default" href="https://discuss.circleci.com/t/a-more-scalable-container-friendly-self-hosted-runner-container-agent-now-in-open-preview/45094">Discuss の投稿 (英語)</a> をご覧ください。
+NOTE: 現在、よりスケーラブルでコンテナフレンドリーなセルフホストランナー、<<container-runner#,コンテナランナー>> の **オープンプレビュー** を行っています。 詳細については、 link:https://discuss.circleci.com/t/a-more-scalable-container-friendly-self-hosted-runner-container-agent-now-in-open-preview/45094[Discuss の投稿 (英語)] をご覧ください。
```
Comment (TangoEnSkai) [inline jekyll/_cci2_ja/runner-overview.adoc:19]: the link is uninteionally exposed when I checked it with `display with the rich diff` as below: 

> よりスケーラブルでコンテナフレンドリーなセルフホストランナー、<a data-type="xref" href="container-runner.html">コンテナランナー</a>の<strong>オープンプレビュー</strong>を行っています。 詳細については、<a data-type="default" href="[https://discuss.circleci.com/t/a-more-scalable-container-friendly-self-hosted-runner-container-agent-now-in-open-preview/45094">Discuss](https://discuss.circleci.com/t/a-more-scalable-container-friendly-self-hosted-runner-container-agent-now-in-open-preview/45094%22%3EDiscuss) の投稿 (英語)</a> をご覧ください。

File: jekyll/_cci2_ja/ecs-ecr.md [hunk 2]
```diff
@@ -118,9 +118,9 @@ workflows:
           tag: "${CIRCLE_SHA1}"
       - aws-ecs/deploy-service-update:
           requires:
-            - aws-ecr/build-and-push-image # このジョブは aws-ecr/build-and-push-image が成功した場合のみ実行されます
+            - aws-ecr/build-and-push-image # only run this job once aws-ecr/build-and-push-image has completed
```
Comment (TangoEnSkai) [inline jekyll/_cci2_ja/ecs-ecr.md:121]: note: I think this is intentional to keep comments in English for codes (e.g. yml)? 
but some have Japanese comments https://circleci.com/docs/ja/hello-world

please confirm which language we want to use for comments.

**Review by TangoEnSkai — COMMENTED** [review]
left some minor comments and queistons but enitre translation LGTM.

File: jekyll/_cci2_ja/runner-overview.adoc (hunk 1 above)
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/runner-overview.adoc:19]: Thanks! The rich diff doesn't seem to recognize asciidoc syntax. I previewed this page locally and the link worked correctly 😃 
スクリーンショット 2022-09-01 9 28 06

File: jekyll/_cci2_ja/ecs-ecr.md (hunk 2 above)
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/ecs-ecr.md:121]: Sorry for poor explanation, we decided not to translate code comments anymore.
https://circleci.slack.com/archives/CJF173R4H/p1644887332807559
So when changes are made to the code, code comments that were previously translated to Japanese may be reverted to English, but this is allowed.

File: jekyll/_cci2_ja/api-developers-guide.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ version:
   - クラウド
 ---
 
-この*API 開発者向けガイド*では、開発者の方々が迅速かつ簡単に CircleCI サービスへの API 呼び出しを行い、ユーザー、パイプライン、プロジェクト、ワークフローに関する詳細情報を返す方法を紹介します。 API v2 の仕様については、[リファレンスドキュメント]({{site.baseurl}}/api/v2)をご覧ください。
+この*API 開発者向けガイド*では、開発者の皆様が迅速かつ簡単に CircleCI サービスへの API 呼び出しを行い、ユーザー、パイプライン、プロジェクト、ワークフローに関する詳細情報を返すための方法を紹介します。 API v2 の仕様については、[リファレンスドキュメント]({{site.baseurl}}/api/v2)をご覧ください。
```
[the remark this answers is not in the document]
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/api-developers-guide.md:13]: Good catch! Somehow this is rendered as `https://circleci.com/docs/ja/api/v2/`. I changed to use absolute path.

File: jekyll/_cci2_ja/updating-server-replicated-version.adoc [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
-= Updating Replicated
+= Replicated の更新
 :page-layout: classic-docs
 :page-liquid:
 :icons: font
 :toc: macro
 :toc-title:
 
-To update your CircleCI server installation, see our https://circleci.com/docs/updating-server/#section=server-administration[Upgrade Guide]. This guide only runs through the steps required to update Replicated, not the CircleCI application.
+CircleCI Server の更新については、https://circleci.com/docs/ja/updating-server/#section=server-administration[更新ガイド]を参照してください。 このガイドでは CircleCI アプリケーションではなく、Replicated の更新に必要な手順について説明します。
```
[the remark this answers is not in the document]
Comment (Mayuko-Sa) [reply jekyll/_cci2_ja/updating-server-replicated-version.adoc]: Good catch! Fixed!

**Comment by Mayuko-Sa** [comment]
@TangoEnSkai 
Thank you for your review! Fixed and commented. Please approve if it is okay 🙇‍♀️

**Comment by Mayuko-Sa** [comment]
I removed `deploying-ios.md` based on this conversation
https://circleci.slack.com/archives/C0289D3QY3E/p1661986339998029

File: jekyll/_cci2_ja/ecs-ecr.md (hunk 2 above)
Comment (TangoEnSkai) [reply jekyll/_cci2_ja/ecs-ecr.md:121]: thank for sharing the background. 
I understood the current rule, and agree to be consistent is the most important! ✅

File: jekyll/_cci2_ja/runner-overview.adoc (hunk 1 above)
Comment (TangoEnSkai) [reply jekyll/_cci2_ja/runner-overview.adoc:19]: ✅ thanks for the confirmation! LGTM

File: jekyll/_cci2_ja/updating-server-replicated-version.adoc (hunk 4 above)
[the remark this answers is not in the document]
Comment (TangoEnSkai) [reply jekyll/_cci2_ja/updating-server-replicated-version.adoc]: ✅ fixed as 4f25cfd882112ad0ffa42c87ef85a9a9561f9c77

File: jekyll/_cci2_ja/api-developers-guide.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (TangoEnSkai) [reply jekyll/_cci2_ja/api-developers-guide.md]: ✅ 4f25cfd882112ad0ffa42c87ef85a9a9561f9c77 fixed it. thx

**Review by TangoEnSkai — APPROVED** [review]
LGTM 👍

**Pull request merged**